docs/topics/ref_glossary.adoc

```diff
@@ -4,7 +4,7 @@
 
 == Product and project names
 
-{name-launcher-oso}:: link:{link-launcher-oso}[{name-launcher-oso}] is a standalone getting started experience offered by Red Hat for jumpstarting cloud-native application development on OpenShift. It provides a hassle-free way of creating functional example applications, called missions, as well as an easy way to build and deploy those missions to OpenShift. 
+{name-launcher-oso}:: link:{link-launcher-oso}[{name-launcher-oso}] is a standalone getting started experience offered by Red Hat for jumpstarting cloud-native application development on OpenShift. It provides a hassle-free way of creating functional example applications as well as an easy way to build and deploy these example applications on OpenShift. 
```

Comment:
Can we avoid words like "hassle-free"? It sounds more like marketing lingo. Also, the sentence makes absolute sense without that.